lib/net.js

```diff
@@ -1075,16 +1076,17 @@ function Server(options, connectionListener) {
   this._connections = 0;
 
   Object.defineProperty(this, 'connections', {
-    get: util.deprecate(function() {
+    get: internalUtil.deprecate(function() {
 
       if (self._usingSlaves) {
         return null;
       }
       return self._connections;
-    }, 'connections property is deprecated. Use getConnections() method'),
-    set: util.deprecate(function(val) {
+    }, 'Server.connections property is deprecated. ' +
+       'Use Server.getConnections() method'),
```

Comment:
For consistent style, I'd recommend `x is deprecated. Use y instead.`. This applies to a few messages here.